### PR TITLE
Add cross compile to aarch64, armhf, and armel

### PR DESCRIPTION
**Pull request by abalmos**
Needed for nestybox/sysbox#360

### CODE REVIEW

**Comment by abalmos** [comment]
Force push was due to missing signoffs

**Review by rodnymolina — CHANGES_REQUESTED** [review]
LG. Just have a couple of minor suggestions.

File: Makefile [hunk 1]
```diff
@@ -31,16 +37,34 @@ LDFLAGS := '-X "main.edition=${EDITION}" -X main.version=${VERSION} \
...
-sysbox-fs-debug: $(SYSFS_SRC) $(SYSIPC_SRC) $(LIBSECCOMP_SRC) $(LIBPIDMON_SRC) $(NSENTER_SRC)
-	$(GO) build -gcflags="all=-N -l" -ldflags ${LDFLAGS} -o sysbox-fs ./cmd/sysbox-fs
+.DEFAULT: sysbox-fs
 
-sysbox-fs-static: $(SYSFS_SRC) $(SYSIPC_SRC) $(LIBSECCOMP_SRC) $(LIBPIDMON_SRC) $(NSENTER_SRC)
-	CGO_ENABLED=1 $(GO) build -tags "netgo osusergo static_build" \
+$(SYSFS_BUILDDIR)/$(SYSFS_TARGET): $(SYSFS_SRC) $(SYSIPC_SRC) $(LIBSECCOMP_SRC) $(LIBPIDMON_SRC) $(NSENTER_SRC)
+	$(GO_XCOMPILE) $(GO) build -ldflags ${LDFLAGS}	-o $(SYSFS_BUILDDIR)/sysbox-fs ./cmd/sysbox-fs
+
+sysbox-fs: $(SYSFS_BUILDDIR)/$(SYSFS_TARGET)
```
[the remark this answers is not in the document]
Comment (abalmos) [reply Makefile]: Fixed the lint

File: Makefile [hunk 2]
```diff
@@ -31,16 +37,34 @@ LDFLAGS := '-X "main.edition=${EDITION}" -X main.version=${VERSION} \
...
+
+$(SYSFS_BUILDDIR)/$(SYSFS_DEBUG_TARGET): $(SYSFS_SRC) $(SYSIPC_SRC) $(LIBSECCOMP_SRC) $(LIBPIDMON_SRC) $(NSENTER_SRC)
+	$(GO_XCOMPILE) $(GO) build -gcflags="all=-N -l" -ldflags ${LDFLAGS} -o $(SYSFS_BUILDDIR)/sysbox-fs ./cmd/sysbox-fs
+
+sysbox-fs-debug: $(SYSFS_BUILDDIR)/$(SYSFS_DEBUG_ TARGET)
```
[the remark this answers is not in the document]
Comment (abalmos) [reply Makefile]: Good catch, thanks!

**Review by rodnymolina — APPROVED** [review]
LG

**Comment by rodnymolina** [comment]
@abalmos, please check [this](https://github.com/nestybox/sysbox-fs/pull/51/checks?check_run_id=3127967936) out to meet the DCO requirement.

**Pull request merged**